### DISCUSSION TITLE
Regarding Future Lego Unverse's Current, Unfortunate State

### DISCUSSION THREAD

**Discussion by AugustRenders** [DarkflameUniverse/DarkflameServer#1136] [Ideas]
As a critic, I'm going to be fully honest, here. I enjoyed playing Lego Universe as a kid, but playing it as an adult who has grown to critique game design, I am not satisfied with the gameplay. There are several bugs, fighting enemies can be frustrating, and some parts of the game feel unfinished or in need of updates for a modern audience.

Lego Universe will likely not progress towards a bright future if it stays in this current state. I believe that with the help of modders in the Lego Universe community, we can change things for the better. What we need is a way to study all of the technical aspects of the game in order to add new content and other changes. If we had some sort of Lego Universe level-editing software, this would be possible. Just as fans of Spongebob: Battle for Bikini Bottom made a level editor (known as Industrial Park,) I feel it might be possible to do something similar with Lego Universe.

In addition to this, we might also need some sort of "Nexus Launcher" that players could use to download mods from a private server and into their client. Basically, something that would match their version of the game with that of the private server in a simple, easy manner.

The Lego Universe community will not last forever if it doesn't find a way to update or fix the game. If there are any modders or software developers out there who wish to keep the game alive and get it more attention with new content, please let me know. And let us collaborate to make Lego Universe a better place!

**Comment by Jettford** [comment]
Hello!

Glad you are being honest, no reason to not be and thanks for the feedback however it isn't really something for us to take on board or to talk about here. DLUs goals simply revolve around restoring the great experiences we enjoyed as kids. While that may not be as enjoyable now as it once was, it restores what once was and preserves it for the future. That is why we have done what we have. Now I am sure you have things you would like to change and to improve, would be awesome to see what you can come up with however here is not the place to discuss that simply because of the nature of our goals.

If you want somewhere to head where you can talk about this kind of thing head to this discord: `https://discord.gg/5A2tcUkS`

You will find like minded people and it is a better place to talk about modding and the future of the game.

**Reply by AugustRenders** [reply]
Thank you very much for guiding me to this, and I'd like to apologize if I made any statements that seemed... out of place, for lack of a better description. I didn't know of any other place where LU fans could interact. GitHub was the only place I could think of, so thanks again. And good day.

**Comment by MustacheMan2** [comment]
I'll be blunt, no one is gonna drop everything to try and get this game on the map. I simply don't see it possible for this game to gain a larger audience than it has, especially since the game as it was back then was not super popular to begin with. There already is a level editor if you can find it, and as someone who used to use it constantly I can tell you that it's good fun, but buggy, and limiting if you don't spend a great deal of time delving into modding as a whole. I'm still wishing there will be other mod tools out someday but simply put no one wants to make the time to make them. (Funny that you mention a Nexus Launcher, because one already exists with that same name, but in its state it's only really useful for launching the game and having multiple server ips to swap out) 

Personally I, along with many others, am still satisfied with the game; it has issues but sometimes those issues give it charm and I've just learned to deal with them. I think this game would have thrived during the pandemic, but there is a near zero chance that LEGO will revive this game in any way, which would be what brings this game back into the light (that or a popular streamer was convinced to play it). 

On a somewhat similar note, my main concern about LU's future is it's compatibility. Maybe I'm just overthinking this, but because of the setup process for the server and its dependencies I fear if contributions ceased today that it wouldn't work on a computer 10 years later. Some of you might roll your eyes at the thought but a release with an executable file like what Uchu did would be quite wonderful, but I won't stress anyone about that, but possibly a consideration?
